doc/guides/west/sign.rst

```diff
@@ -41,12 +41,17 @@ Some additional notes follow. See ``west sign -h`` for detailed help.
   .. code-block:: console
 
      west sign -t imgtool -B my-signed.bin -H my-signed.hex [...]
-- For reference, here is an example for how to build :ref:`hello_world` for
-  MCUboot using ``west build``:
 
-  .. code-block:: console
+Example build flow
+******************
+For reference, here is an example showing how to build :ref:`hello_world` for
+MCUboot using ``west``:
+
+.. code-block:: console
 
-     west build -b YOUR_BOARD zephyr/samples/hello_world -- -DCONFIG_BOOTLOADER_MCUBOOT=y
+   west build -b YOUR_BOARD samples/hello_world -- -DCONFIG_BOOTLOADER_MCUBOOT=y
+   west sign -t imgtool -- --key YOUR_SIGNING_KEY.pem
+   west flash --bin-file zephyr.signed.bin
```

Comment:
Won't this overwrite mcuboot itself? Flashing bin files with west is a bit harder to do right now and involves command line options that I'd prefer to deprecate (like `--dt-flash`).

Reply:
I tested it with jlink runner and it works fine. If however flashing .bin files is not supported by other runners I may modify it to read
```
   west flash --hex-file zephyr.signed.hex
```
Or should I better close this PR for now? I was simply missing an example of the full west build flow and usually referring to #12903 initial comment as documentation.

Reply:
If you can modify it to refer to hex files only, and add a line saying that the availability of a hex file depends on your build configuration, that would be wonderful and I'd be happy to +1 this. Thanks!

Reply:
Fixed